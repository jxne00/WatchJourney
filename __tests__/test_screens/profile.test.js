import React from 'react';
import renderer from 'react-test-renderer';

import ProfileScreen from '../../screens/profile/Profile';

describe('tests for Profile screen', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<ProfileScreen />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('has 1 child', () => {
    const tree = renderer.create(<ProfileScreen />).toJSON();
    expect(tree.children.length).toBe(1);
  });
});

