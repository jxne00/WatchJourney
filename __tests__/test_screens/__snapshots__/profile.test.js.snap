// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests for Profile screen renders correctly 1`] = `
<RCTSafeAreaView
  style={
    {
      "alignItems": "center",
      "backgroundColor": "#74369D",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <View
    style={
      {
        "alignItems": "center",
        "backgroundColor": "#74369D",
        "flex": 1,
        "justifyContent": "center",
      }
    }
  >
    <Text
      style={
        {
          "fontFamily": "Poppins-Regular",
          "fontSize": 16,
        }
      }
    >
      This is the Settings page
    </Text>
  </View>
</RCTSafeAreaView>
`;
